CMakeLists.txt

```diff
@@ -296,9 +296,10 @@ target_include_directories(crc32c
     $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
 )
 
-target_compile_definitions(crc32c
-PRIVATE
-  CRC32C_HAVE_CONFIG_H=1
+set_property(
+  TARGET crc32c_arm64 crc32c_sse42 crc32c
```

Comment:
If you want to define it for all targets, is it necessary to list all of them? There's no "blanket" option to just define it?

Reply:
Yes, it is possible to `add_definitions(-DCRC32C_HAVE_CONFIG_H=1)`. It works for all targets in this directory, even those which do not require  such a definition, e.g., `crc32c_bench`. But it is not relevant to the Bitcoin Core :)

Do you want me apply this solution?

Reply:
I'm not sure. I admit I don't have a good view on the compromises involved.

Reply:
It is considered best CMake practice to keep properties bound to targets.